apps/publications/src/app/dialogs/add-authors-dialog/add-authors-dialog.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="user-theme">
+  <h1 mat-dialog-title>{{'DIALOGS.ADD_AUTHORS.TITLE' | translate}}</h1>
+  <div mat-dialog-content>
+    <mat-form-field class="mr-2 adjust-width">
+      <label>
+        <input matInput
+               autocomplete="false"
+               placeholder="{{'DIALOGS.ADD_AUTHORS.SEARCH_PLACEHOLDER' | translate}}"
+               (keyup.enter)="onSearchByString()"
+               [formControl]="searchControl">
+      </label>
+      <mat-error>
+        {{'DIALOGS.ADD_AUTHORS.EMPTY_SEARCH_MESSAGE' | translate }}
+      </mat-error>
+    </mat-form-field>
+    <button mat-flat-button
+            color="accent"
+            class="mr-2"
+            [disabled]="loading || searchControl.value.trim() === ''"
+            (click)="onSearchByString()">
+      <mat-icon>search</mat-icon>
+    </button>
+    <div *ngIf="!loading">
+      <perun-web-apps-authors-list
+        *ngIf="firstSearchDone"
+        [authors]="authors"
+        (addAuthor)="addAuthor($event)"
+        [disableRouting]="true"
+        [pageSize]="pageSizeAuthors"
+        [displayedColumns]="['id', 'name', 'organization', 'email', 'add']"
+        (page)="pageChangedAuthors($event)">
+      </perun-web-apps-authors-list>
+      <app-alert *ngIf="!firstSearchDone" [alert_type]="'info'">
+        {{'DIALOGS.ADD_AUTHORS.SEARCH_INFO' | translate}}
+      </app-alert>
+    </div>
+  </div>
+  <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+  <div mat-dialog-content>
+    <div *ngIf="firstSearchDone">
```

Comment:
I think that Authors to add can be showed even before some searching happens - but thats just suggestion